src/Strategy/Python/Pipenv.hs

```diff
@@ -128,7 +128,7 @@ buildNodes PipfileLock{..} = do
                -> PipfileDep
                -> m ()
   addWithEnv env sourcesMap depName dep = do
-    let pkg = PipPkg depName (T.drop 2 (fileDepVersion dep))
+    let pkg = PipPkg depName $ T.drop 2 <$> fileDepVersion dep
```

Comment:
[nit]: Not sure this `$` is the most readable due to the `<$>` operator.  I'll let you decide, but I think it might be more readable as this.
```suggestion
    let pkg = PipPkg depName (T.drop 2 <$> fileDepVersion dep)
```
That would prevent confusion about which operator applies where, which was a little confusing to me at first.  Normally, I prefer to use `.` and `$` liberally.  However, I think it's less readable in the presence of other infix operators.

Reply:
Yea I see what you're saying here, I agree. Thanks.